datadog-service/0.15.0/README.md

```diff
@@ -0,0 +1,198 @@
+# datadog-service
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/keptn-sandbox/datadog-service)
+[![Go Report Card](https://goreportcard.com/badge/github.com/keptn-sandbox/datadog-service)](https://goreportcard.com/report/github.com/keptn-sandbox/datadog-service)
+
+This implements a datadog-service for Keptn. If you want to learn more about Keptn visit us on [keptn.sh](https://keptn.sh)
+
+Check the issue on the main repo for more info: https://github.com/keptn/keptn/issues/2652
+
+
+## Quickstart
+If you are on Mac or Linux, you can use [examples/kup.sh](https://raw.githubusercontent.com/keptn-sandbox/datadog-service/release-0.15.0/examples/kup.sh). This script creates a local minikube cluster, installs Keptn, Istio, Datadog and the Datadog integration for Keptn (check the script for pre-requisites). 
+
+To use the script,
+```bash
+$ export DD_API_KEY="<your-datadog-api-key>" DD_APP_KEY="<your-datadog-app-key>" DD_SITE="datadoghq.com" 
+$ examples/kup.sh
+```
+Check [the official docs](https://docs.datadoghq.com/account_management/api-app-keys/) for how to create Datadog API key and Application key
```

Comment:
```suggestion
Check [the official docs](https://docs.datadoghq.com/account_management/api-app-keys/) for how to create the Datadog API key and Application key
```

Reply:
Corrected. Thank you for the suggestion!